alica_engine/src/engine/teammanager/TeamManager.cpp

```diff
@@ -113,8 +113,7 @@ void TeamManager::readSelfFromConfig(const YAML::Node& config)
     _localAnnouncement.token = rd();
     _localAnnouncement.senderName = localAgentName;
     _localAnnouncement.senderSdk = _engine->getVersion();
-    // TODO: add plan hash
-    _localAnnouncement.planHash = 0;
+    _localAnnouncement.planHash = _engine->getMasterPlanId();
```

Comment:
planHash being the master plan id seems unintuitive.  Perhaps we should change the name of the field in the msg?

Reply:
Its ok for now. Was supposed to be a hash of all the plans, behaviours combined but this could be extended later.

Reply:
Yeah just from the field name I would think maybe it's the hash of the current active plan or something